doc/cprover-manual/properties.md

```diff
@@ -203,20 +203,22 @@ goto-instrument error_example.goto error_example_replaced.goto \
 cbmc error_example_replaced.goto
 ```
 
-Which gets us the output
-
-> ** Results:
-> error_example.c function api_error
-> [api_error.assertion.1] line 4 assertion false: FAILURE
->
-> error_example.c function internal_error
-> [internal_error.assertion.1] line 5 assertion false: FAILURE
->
-> ** 2 of 2 failed (2 iterations)
-> VERIFICATION FAILED
-
-As opposed to the verification success we would have seen without the
-instrumentation step.
+This generates the following output:
+
+```
+** Results:
+error_example.c function api_error
+[api_error.assertion.1] line 4 assertion false: FAILURE
+
+error_example.c function internal_error
+[internal_error.assertion.1] line 5 assertion false: FAILURE
+
+** 2 of 2 failed (2 iterations)
+VERIFICATION FAILED
+```
+
+Without the instrumentation step we would have seen
+"verification successful".
```

Comment:
I think this should match the output precisely, i.e., be all-caps.

Reply:
done